testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -403,15 +407,14 @@ fn generate_new_era() -> String {
 
 pub async fn install_testnet_resources(
     kube_namespace: String,
-    base_num_validators: usize,
-    base_validator_image_tag: String,
-    base_genesis_image_tag: String,
+    num_validators: usize,
+    num_fullnodes: usize,
+    validator_image_tag: String,
```

Comment:
I am assuming this is used for both validator and FN, in that case, let's rename this to `node_image_tag` ?